### ISSUE TITLE
Standalone scummvm version and the virtual mouse pointer

### ISSUE THREAD

**Issue by mariocron** [rg35xx-cfw/Koriki#13]
### What happened?

First the reason why I think my request is important and why I consider it a bug:
The default option to run scummvm games (Home Computers -> Scumm Vm -> games list) uses the libretro version of scummvm, this version has much worst performance than the standalone version of scummvm which can be accessed from Apps & Games -> APPS -> SCUMMVM (for example, Grim Fandango is unplayable in the libretro version, but runs perfectly smooth in the standalone version - checked using Batocera firmware for RG35xx). 
The bug is that in the standalone version of scummvm the D-pad doesn't move the virtual mouse pointer, so you can't launch/add/configure games. I've spend several hours trying to remap the buttons in scummvm.ini (system/.config/scummvm.ini) without success - it seems some keymap_XXX configuration settings don't affect how scummvm works. Same bug is related also to the Batocera firmware for RG35xx, but in Batocera choosing scummvm game from the list runs the standalone version of scummvm, so at least you can start a game. 

To sum up:
- the D-pad doesn't move the virtual mouse pointer in the standalone version of scummvm
- buttons cannot be remapped (which is also important due to the fact that several games use the D-pad to move the character, not to move the mouse pointer)

Please consider changing the default scummvm engine to the standalone version whose perfomance really shines over the libretro version and unleash the power of RG35xx (for ex. on miyoo mini running Onion OS Grim Fandango is unplayable and Blade Runner is playable only whe you reduce framerate to 15)

**Comment by acmeplus** [comment]
Yes, it's definitely planned to support that. Progress will be posted in this issue, but most likely won't be ready for koriki 1.0
